resources/js/components/modules/data/modules/alcoholModule.js

```diff
@@ -1103,7 +1111,7 @@ export const alcoholModuleSlides = {
       paragraph: {
         0: 'Now is a great time to set a wellness goal to practice responsible drinking, or just  improve your overall health!'
       },
-      textAreaPlaceHolder: 'For example, \"Next week I will plan to drink one fewer drink than normal when I go out with friends." Or, \"I\'ll work out at least once next week!\"'
+      textAreaPlaceHolder: 'For example, "Next week I will plan to drink one fewer drink than normal when I go out with friends." Or, "I\'ll work out at least once next week!"'
```

Comment:
Off topic but when should we use escape characters for ours strings?

Reply:
Say you have 'It's' , you need to escape 'It\'s' because they are using the same quotations.